lib/scrabble.rb

```diff
@@ -1,18 +1,45 @@
+require_relative 'dictionary'
+
 class Scrabble
+  include Dictionary 
+
+  attr_accessor :score
+  
+  def initialize 
+    @score = 0
+  end
 
   def score(word)
-    1
+    return @score = 0 if word.nil? || word.empty? 
+    reset_score
+    plus_ten_if_over_seven(word)
+    formatted = format(word)
+    formatted.each {|letter| @score += point_values[letter]}
```

Comment:
You could just do format(word).each to shorten this mehtod; we'd also love to see you pushing yourself to use a more advanced enum like reduce, but not expecting it at this point.